wiki/Help_Center/en.md

```diff
@@ -145,6 +145,20 @@ If you have an active supporter tag, you can see all maps you have submitted a s
 
 Unfortunately, we do not keep track of all maps a user has downloaded, so it is impossible to get a list of them. The closest you could be to it would be using the aforementioned filter to recover at least the maps you've played before.
 
+### I do not receive any emails from osu!/I've forgotten my email address!
```

Comment:
change to `### I didn't receive any emails from osu!`
underneath, put `_This could also be phrased, "I forgot my email address!"_`

---

Otherwise, then just change `I do not receive` to `I didn't receive`